saving-color-elements/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "my-project",
+    "version": "0.0.0",
+    "dependencies": {
+        "node-fetch": "^3.0.0"
+    }
+}
```

Comment:
Do we need a `package.json` for this sample? Taking a (maybe too?) quick look at it, it looks like a static HTML page?